keyboards/mechlovin/olly/jf/config.h

```diff
@@ -72,3 +72,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 /* Debounce reduces chatter (unintended double-presses) - set 0 if debouncing is not needed */
 #define DEBOUNCE 5
+
+/* Bootmagic Lite key configuration */
+#define BOOTMAGIC_LITE_ROW 1
+#define BOOTMAGIC_LITE_COLUMN 14
```

Comment:
Based on this, [keyboards/mechlovin/olly/jf/readme.md#L23](https://github.com/qmk/qmk_firmware/blame/2b4eee4bf75f828360c4d874b3b6e0b4afa10101/keyboards/mechlovin/olly/jf/readme.md#L23) should read:

```
By bootloader: hold Backspace key while plugging in
```